### ISSUE TITLE
Problem on associative recall

### ISSUE THREAD

**Issue by xiaoshingshing** [MarkPKCollier/NeuralTuringMachine#8]
Using ntm as the default setting works fine on copy task, but it doesn't work on associative recall task.

I run for more than 20000 training steps with 5 hours and 1 GPU, and the sequence error doesn't drop.

Does associative recall task needs another setting?

**Comment by MarkPKCollier** [comment]
What are you setting `max_seq_len` and `num_bits_per_vector` to? To replicate the results in the paper `max_seq_len=6` (that's 6 items so 18 vectors) and `num_bits_per_vector=6`

**Comment by xiaoshingshing** [comment]
I will try the above configuration.

By the way, I run the previous configuration for the whole weekends, and the training process reach 350000 training step, and the 'nan' loss occur. I have stopped the previous configuration.
